files/app/authenticators/oauth2-gavant.ts

```diff
@@ -4,23 +4,36 @@ import { assign } from '@ember/polyfills';
 import { cancel, later } from '@ember/runloop';
 import { inject as service } from '@ember/service';
 import { isEmpty } from '@ember/utils';
+import { cached } from '@glimmer/tracking';
 
 import Oauth2PasswordGrantAuthenticator from 'ember-simple-auth/authenticators/oauth2-password-grant';
 import { SessionAuthenticatedData } from 'ember-simple-auth/services/session';
 import isFastBoot from 'ember-simple-auth/utils/is-fastboot';
 
-import { ServerErrorPayload } from '<%= modulePrefix %>';
 import ENV from '<%= modulePrefix %>/config/environment';
 import AjaxService from '<%= modulePrefix %>/services/ajax';
+import { fastbootSafeBtoa } from '<%= modulePrefix %>/utils/encoding';
 
 export default class Oauth2GavantAuthenticator extends Oauth2PasswordGrantAuthenticator {
     @service declare ajax: AjaxService;
 
     serverTokenEndpoint = `${ENV.apiBaseUrl}/oauth2/token`;
     serverTokenRevocationEndpoint = `${ENV.apiBaseUrl}/oauth2/logout`;
+    serverTokenRefreshEndpoint = `${ENV.apiBaseUrl}/oauth2/refresh`;
+
+    @cached
+    get serverBasicAuthorization() {
+        return `Basic ${fastbootSafeBtoa(`${ENV.clientId}:${ENV.clientSecret}`)}`;
+    }
```

Comment:
@napafundi I might be wrong, but I thought this stuff (the Basic Authentication header, etc) was only needed for the Stein app specifically. Do you know if this is something we are adding as "standard" for all new apps moving forward?